content/doc/addons/postgresql/_index.md

```diff
@@ -127,6 +127,10 @@ pgtap       | Unit testing for PostgreSQL
 pgvector    | Vector data type and ivfflat and hnsw access methods
 timescaledb | Enables scalable inserts and complex queries for time-series data (Community Edition)
 
+{{< callout type="warning" >}}
+On-demand extensions are not available for DEV addons.
```

Comment:
```suggestion
On-demand extensions are not available for DEV plans.
```

Reply:
Also, maybe use "aren't" like the bot said? :shrug:

Reply:
Yeah, we unified the term. It's `add-on` and it flags `addon`

```suggestion
On-demand extensions aren't available for DEV plans.
```